Uploading Task 1

witalikh opened the pull request.

witalikh committed:
Uploading Task 1

tetianastoyko commented on Task 1.py:
```diff
@@ -71,7 +71,8 @@ def main():
             if count is 0:
                 break
-            
+
```
Програма не поділена на логічні частини - функції, що вимагалось і озвучувалось на парі
Для отримання унікальних значень не повинні були використовуватись вбудовані методи, тоді ваше завдання простіше, ніж у інших.

witalikh replied:
Перепрошую, момент з вбудованими методами я прослухав. Дякую за зауваження

witalikh committed:
Updating structure and logic according to demands

* Refactoring main function:
 - input an array
 - calculate the unique elements
* Another implementation of finding unique elements
Based on comparison adjacent elements in sorted list
 - created quick sort algorithm for such things
* Removed cheating methods from algorithms :/
